### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by hitman-11**
Week 1 Assignment HTML Page

### CODE REVIEW

File: Week-1/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <div class="sidebar">
+    <i class="las la-bars"></i>
+    <i class="las la-home"></i>
+    <i class="las la-search"></i>
+    <i class="las la-volume-up"></i>
+    <i class="las la-user"></i>
+    <i class="las la-cog"></i>
+    <i class="lab la-spotify"></i>
+    <i class="las la-cloud"></i>
+
```
Comment (rahul-pesto) [inline Week-1/index.html]: code should be formatted before raising the PR

improper indentation in both HTML and CSS file

File: Week-1/.readme [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (rahul-pesto) [inline Week-1/.readme]: why is this file empty?

**Review by rahul-pesto — CHANGES_REQUESTED** [review]
Assignment not completed!

please complete both parts of the assignment
Exercise 1: Create a readme file that explains browser functionality.
Exercise 2: Clone of mock-up which was provided, done in the HTML and CSS files.

**Comment by hitman-11** [comment]
Changes Made according to your PR comments. The new page looks like this.
Screenshot (1)

**Review by rahul-pesto — APPROVED** [review]

**Pull request merged**